docs/graphql/manual/queries/query-filters.rst

```diff
@@ -187,6 +187,35 @@ Fetch a list of articles that were published on a certain date (``published_on``
       }
     }
 
+Example: Column
+^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The underline should match the title length

Reply:
will fix

Reply:
Sorry I'm new. I pushed the changes, but how do I add it to this PR?